bootstrap/mocks/things.go

```diff
@@ -166,7 +166,7 @@ func (svc *mainfluxThings) UpdateKey(string, string, string) error {
 	panic("not implemented")
 }
 
-func (svc *mainfluxThings) ListThings(string, uint64, uint64) (things.ThingsPage, error) {
+func (svc *mainfluxThings) ListThings(string, uint64, uint64, string) (things.ThingsPage, error) {
```

Comment:
Consider using variadic functions so that we do not have to pass this param every time.

I am not saying it is better, just analyze this possibility.

@anovakovic01 @dusanb94 @nmarcetic comments?

Reply:
Instead of using variadic functions and checking the variables like:
```go
package main

import (
	"fmt"
	"reflect"
)

func test(a ...interface{}) {
	for i, param := range a {
		fmt.Printf("%d : %s\n", i, reflect.TypeOf(param))
	}
}

func main() {
	test(1, "my_way", true)
}
```
It might be better if we use some dependency injection framework like [wire](https://github.com/google/wire) ([blog post](https://blog.golang.org/wire))or one from awesome-go's [di list](https://github.com/avelino/awesome-go#dependency-injection). I've used DI in other languages, but not in Go and I can not say which framework is better.

Reply:
I agree with @anovakovic01